packages/eslint-config-vivy/index.js

```diff
@@ -1,19 +1,24 @@
 module.exports = {
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   extends: [
     'airbnb',
     'prettier',
     'prettier/react',
-    "plugin:jest/recommended"
-],
-  "plugins": [
-    "prettier",
-    "jest"
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
   ],
+  plugins: ['prettier', 'jest', 'react'],
```

Comment:
no `@typescript-eslint` ? 😕

Reply:
I have a huge list of plugins I would like to add but I think this should be discussed later this PR fixed some issues

Reply:
I thought that we have to use it that's that's why 😉